adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/internal/service/ParametersValidationService.java

```diff
@@ -0,0 +1,43 @@
+package de.adorsys.xs2a.adapter.service.ing.internal.service;
+
+import de.adorsys.xs2a.adapter.service.Oauth2Service;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParametersValidationService {
+
+    private static final String SCOPE = Oauth2Service.Parameters.SCOPE;
+    private static final String DEFAULT_SCOPE = "payment-accounts:transactions:view";
+    // todo add 'payment-accounts:orders:create' scope when PIS flow will be implemented
+    private static final List<String> availableScopes = Arrays.asList(DEFAULT_SCOPE, "payment-accounts:balances:view");
```

Comment:
As far as I see, this collection is used to check whether it contains some String value or not. Seems like an ideal case for using HashSet to improve the performance. What do you think?

Reply:
It's probably gonna be only three elements there so not much performance improves, no problem with replacing with Set though.